controllers/profiles/reconciler_dev.go

```diff
@@ -51,12 +56,46 @@ const (
 	recoverDeploymentErrorRetries = 3
 	// recoverDeploymentErrorInterval interval between recovering from failures
 	recoverDeploymentErrorInterval = 5 * time.Minute
+	extResPrefix                   = "sw.kogito.kie.org/resource-"
```

Comment:
Not sure if this prefix makes sense since your suffix already has `resource-`. Also, we already have an annotation for the domain. I think this const is not needed.

Reply:
@ricardozanini we use this prefix when we read all the labels in the worflow to search if someone add external resources to mount from a configmap